### PR TITLE
More diverse launch options

### PR DESCRIPTION
**Pull request by maitovelkkis**
Slightly more diverse launch options solutions which allows custom server ip, selecting game language and potentially more.

### CODE REVIEW

**Comment by PhilippvK** [comment]
Thank you very much. I will check it out over the weekend!

File: client/src/org/moparforia/client/Launcher.java [hunk 1]
```diff
@@ -67,16 +67,30 @@ public static void main(String[] args) throws Exception {
...
+			for(int i = 0;i<args.length ; i++)
+			{
+				if(args[i].equals("-server")){
+					server = args[i+1]; //grabs the next string after -server
+				}
+				if(args[i].equals("-lang")){
+					lang = args[i+1]; //grabs the next string after -lang
+				}
+			}
+			System.out.println(server);
```
Comment (PhilippvK) [inline client/src/org/moparforia/client/Launcher.java:90]: Please remove those 2 lines or make them more verbose than just printing a value.

File: client/src/org/moparforia/client/Launcher.java [hunk 2]
```diff
@@ -67,16 +67,30 @@ public static void main(String[] args) throws Exception {
...
-
-        instance = new Launcher(server);
+		
+			//easily editable optional client launch options, at the moment supports custom server ip and language pack
+			//example: client.jar -server 125.456.789.123 -lang en_US
+			
+			String server = "127.0.0.1"; //dafault server ip
+			String lang = "en_US"; //dafault language package (comes with en_US, fi_FI and sv_SE)
+			
+			if(args.length==1){ //support for "legacy" .bats where only ip provided after jar
```
Comment (PhilippvK) [inline client/src/org/moparforia/client/Launcher.java:77]: Great idea!

File: client/src/org/moparforia/client/Launcher.java [hunk 3]
```diff
@@ -67,16 +67,30 @@ public static void main(String[] args) throws Exception {
...
+				server=args[0];
+			}
+			
+			for(int i = 0;i<args.length ; i++)
```
Comment (PhilippvK) [inline client/src/org/moparforia/client/Launcher.java:81]: Some error checking would be useful here.

- Out of bounds possible if there is a missing value for an argument
- check if lang has a valid format and maybe give an error message when there is not localization for a given country code
- Optional: server should bei either localhost, an IPv4 Adress or a Domain/Hostname

File: client/src/org/moparforia/client/Launcher.java (hunk 1 above)
Comment (maitovelkkis) [reply client/src/org/moparforia/client/Launcher.java:90]: Aah this println was left in on accident when I was debugging something.

File: client/src/org/moparforia/client/Launcher.java (hunk 2 above)
Comment (maitovelkkis) [reply client/src/org/moparforia/client/Launcher.java:77]: Thanks!

File: client/src/org/moparforia/client/Launcher.java (hunk 3 above)
Comment (maitovelkkis) [reply client/src/org/moparforia/client/Launcher.java]: I have made sure out of bounds shouldn't be possible now... But haven't implemented any kind of format checking at least yet.

**Comment by PhilippvK** [comment]
Great work!

**Pull request merged**